contents/blog/dbt-with-marquez/index.mdx

```diff
@@ -16,29 +16,29 @@ At the time, my defining question was “how can I make all of my most important
 
 I ask questions like:
 * Does [x] technology space matter, and to whom? Is it waxing or waning?
 * Are there adjacent ecosystems we should be collaborating with?
 * Who are the influencers in this space? Who are the major contributors?
 * What challenges are users facing? What does successful adoption look like?
 
-There are a lot of ways to try to answer these questions. None of them are any more reliable than human behavior itself, and every resulting metric requires analysis and judgment. But there are still some pretty fun things to discover. And what better data source to mine to understand technical audiences than [Stack Overflow](https://www.snowflake.com)?
+There are a lot of ways to try to answer these questions. None of them are any more reliable than human behavior itself, and every resulting metric requires analysis and judgment. But there are still some pretty fun things to discover. And what better data source to mine to understand technical audiences than [Stack Overflow](https://www.stackoverflow.com)?
 
 So let’s see what we can learn from the Stack Overflow [public data set in BigQuery](https://cloud.google.com/blog/topics/public-datasets/google-bigquery-public-datasets-now-include-stack-overflow-q-a). But not the whole thing; it is very large, so let’s study just a part of it. I created a [sample dbt project](http://github.com/rossturk/stackostudy/) that contains a handful of models to study all of the questions and answers we can find about the topic of ELT. These models:
 
 * Create slices of the key Stack Overflow tables, pulling them into a separate BigQuery project. These slices only contain the rows that are related to questions tagged with “elt”. That way, we can query them tortuously all day long without scanning through gobs of partitions and running up our bill.
 * Augment these slices by performing some helpful calculations - in this case, the number of upvotes/downvotes per question.
 * Populate two summary tables for consumption by a BI system of some sort: a daily summary table that can be used to study trends and a user summary table that can be used to learn about the most influential contributors.
 
-This is exactly the kind of experiment I have run multiple times over the years, across numerous stacks. It’s usually pretty messy. But this time, after running all of these models, we will be rewarded with a gorgeous Marquez lineage graph. We’ll be able to see how everything fits together.
+This is exactly the kind of experiment I have run multiple times over the years, across numerous stacks. It’s usually pretty messy. But this time, after running all of these models, we will be rewarded with a gorgeous [Marquez](https://marquezproject.github.io/marquez/) lineage graph. We’ll be able to see how everything fits together.
```

Comment:
I would use the [https://marquezproject.ai](https://marquezproject.ai) for Marquez